lms/djangoapps/discussion/tasks.py

```diff
@@ -56,7 +56,22 @@ def send_ace_message(context):
 
 def _should_send_message(context):
     cc_thread_author = cc.User(id=context['thread_author_id'], course_id=context['course_id'])
-    return _is_user_subscribed_to_thread(cc_thread_author, context['thread_id'])
+    return (
+        _is_user_subscribed_to_thread(cc_thread_author, context['thread_id']) and
+        _is_not_subcomment(context['comment_id']) and
+        _is_first_comment(context['comment_id'], context['thread_id'])
+    )
+
+
+def _is_not_subcomment(comment_id):
+    comment = cc.Comment.find(id=comment_id).retrieve()
+    return not getattr(comment, 'parent_id', None)
+
+
+def _is_first_comment(comment_id, thread_id):
+    thread = cc.Thread.find(id=thread_id).retrieve(with_responses=True)
```

Comment:
This is going to get called a lot, and return `False` almost all of the time. Is there any way to limit calls here? Probably not without a migration, right?